front-end/src/App.test.js

```diff
@@ -14,7 +15,9 @@ describe('App', () => {
     );
 
     afterAll(() => {
-        store.dispatch(setTranscriptUploadStatus(false))
+        store.dispatch(setTranscriptUploadStatus(false));
+        store.dispatch(clearAnalyzedTranscript);
```

Comment:
```suggestion
        store.dispatch(clearAnalyzedTranscript());
```
should this have brackets?